docker-compose.yml

```diff
@@ -30,10 +30,11 @@ services:
       - REDIS_HOST=${REDIS_HOST}
       - REDIS_PORT=${REDIS_PORT}
       - ENABLE_REDIS_CACHE=${ENABLE_REDIS_CACHE}
+      - WORKERS=${WORKERS}
     container_name: metadata_api
     tty: true
     command: >
-      bash -c "uvicorn main:app --host 0.0.0.0 --port 8014 --reload --log-level=trace"
+      bash -c "uvicorn main:app --host 0.0.0.0 --port 8014 --workers ${WORKERS} --log-level=trace"
```

Comment:
Unrelated to your changes, but still:
The log level trace for uvicorn is above even debug. Do we need this? I'd recommend to set it lower - e.g. info.
Especially for production work.

Reply:
I think `docker-compose.yaml` is only used to run tha app locally (@kamaldodiya please correct me if I'm wrong), k8s uses the command in `k8s/base/deployment.yaml`